index.js

```diff
@@ -53,15 +53,19 @@ const generateGlobTasks = (patterns, taskOpts) => {
 };
 
 const globDirs = (task, fn) => {
-	if (Array.isArray(task.opts.expandDirectories)) {
-		return fn(task.pattern, {files: task.opts.expandDirectories});
+	const opts = {};
```

Comment:
You could write this function as:

```js
let opts = {cwd: task.opts.cwd};

if (Array.isArray(task.opts.expandDirectories)) {
	opts = Object.assign(opts, {files: task.opts.expandDirectories});
} else if (typeof task.opts.expandDirectories === 'object') {
	opts = Object.assign(opts, task.opts.expandDirectories);
}

return fn(task.pattern, opts);
```

Reply:
`let opts = {cwd: task.opts.cwd};` is clear for me, but I don't understand why would you reassign `opts` variable if `Object.assign` is doing it. Are there any problems with it on some node versions?

Any way, I did what you asked and I replaced `'tmp'`literal with `tmp` variable. Hope you are agree on defining it as a constant.